script/ghIssues.pl

```diff
@@ -96,6 +103,7 @@ sub getIssues{
                 last_update => $issue->{updated_at},
                 last_commit => $last_commit,
                 last_comment => $last_comment,
+                producer => $issue->{assignee}->{login} || undef,
```

Comment:
@jdorweiler Apparently not a lot of PRs have an assignee, I'm getting a lot of undef producer values :(
Maybe fall back to @moollaza instead of undef? Or randomly choose between him and @jagtalon :smile: 

Just throwing ideas around!